docs/device-agent/quickstart.md

```diff
@@ -7,109 +7,59 @@ navOrder: 2
-2. Run the following commands in sequence:
-
-Windows:
-```bash
-mkdir c:\opt\flowfuse-device
-cd c:\opt\flowfuse-device
-npm install -g @flowfuse/device-agent
-flowfuse-device-agent --ui --ui-user admin --ui-pass password --ui-port 8081
-```
-Linux/MacOS:
-```bash
-mkdir /opt/flowfuse-device
-cd /opt/flowfuse-device
-npm install -g @flowfuse/device-agent
-flowfuse-device-agent --ui --ui-user admin --ui-pass password --ui-port 8081
-```
-
-Note: The flags used in the command above have the following meanings:
-- `--ui`: Start the device agents web-based user interface.
-- `--ui-user`: Specify the username for the web user interface.
-- `--ui-pass`: Specify the password for the web user interface.
-- `--ui-port`: Specify the port for the web user interface.
-- You can find more details about these flags [here](https://flowfuse.com/docs/device-agent/running/).
-
-### Add Device in FlowFuse
-1. Open your web browser and go to your FlowFuse application page.
-2. Navigate to the **Devices** section.
-3. Click **Add Device.**
-4. Fill in the **Device Name** and **Device Type,** then click **Add.**
+### Prerequisites
 
-<img src="images/add_device.png" width=500 />
+If you do not already have the device agent installed, follow the [install instructions](./install.md), otherwise, proceed to the next step.
 
-### Configure Device Agent
-1. Expand the Manual setup section, click the **Copy** button below the **Device Configuration**.
+_NOTE: It is good practice to keep your Device Agent up to date. Update instructions are also provided in the link above._
 
-<img src="images/config_yml2.png" width=500 />
+_NOTE: If you do not have access to a terminal, follow the [Quick Start with Web UI](./quickstart-web-ui.md) guide._
 
-2. Open a new browser tab and navigate to the IP address of the device using the port defined when starting up the device agent.  **\<\<ip of device>>:\<\<port>>**, e.g., localhost:8081
-3. Paste the copied device configuration into the **Agent Configuration** field. This configuration contains vital information instructing the device on how to communicate with FlowFuse. It's crucial to keep this information secure and not share it with unauthorized individuals.
+### Step 1: Add a new Device
+1. Open your web browser and go to your FlowFuse application page.
+2. Navigate to the **Devices** section.
+3. Click **Add Device.**
+4. Fill in the **Device Name** & **Device Type** and select an **Application**
 
-<img src="images/device_gui.png" width=500 />
+    <img src="images/add_device.png" />
+6. Click **Add**
+7. You will be presented with a **Device Configuration** that you will need in the next step. _(Do not close this window just yet.)_
 
-4. Click `Apply.`
+    <img src="images/config_yml2a.png"/>
 
-### Link Device to Application
-1. Return to the **Devices** section in your FlowFuse application page.
-2. Locate your newly added device and click its menu.
+### Step 2: Connect the Device
```

Comment:
A need for an explicit pointer to the _"Running Your Agent"_ at the end of this step? This get's it configured, but they can't proceed to "Developer on Device" without having run the agent first.

Reply:
Done in [99a06b1](https://github.com/FlowFuse/flowfuse/pull/3541/commits/99a06b1a6f8466a58026a6af55f1ae26afbfded8)